spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-stream-binder-functional-sample/pom.xml

```diff
@@ -32,4 +32,16 @@
     </dependencies>
   </dependencyManagement>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-maven-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
```

Comment:
What's the effect of this skip configuration on Spring Boot Maven plugin?

Reply:
This sample is a bit funky - the `-sink` and `-source` modules are supposed to be independent, so in theory, they should have been able to be `repackaged` just fine. 

However, the third module (`-sample-test`) only contains tests. The test imports both the -sink and -source modules as _libraries_, which (apparently) don't play well with the Boot `repackaging` goal. 

So, at first I only disabled `spring-boot-maven-plugin` on the `-sample-test`, and it `mvn compile`d, but failed at runtime with a ClassNotFound exception from one of the other modules (or a related exception, I don't remember exactly). To workout the failing test, I just disabled it on the whole thing.